chess_ng/board.py

```diff
@@ -134,6 +137,17 @@ def is_draw_by_repetition(
             return False
         return len(set(self.move_history[-number_of_moves:])) == number_of_teams * 2
 
+    def is_draw_by_fifty_moves(self):
+        if len(self.move_history) < 100:
+            return False
+        last_fifty_moves = self.move_history[-100:]
+        for move in last_fifty_moves:
+            # If there was a pawn move or a capture, it's a draw
+            if isinstance(move[0], Pawn) or move[2]:
```

Comment:
Better to unpack move tuple beforehand to avoid error-prone indexing:
```python
for piece, _, was_captured in last_fifty_moves:
    if isinstance(piece, Pawn) or was_captured:
        ...
```